test_util/test_esmry_lod.cpp

```diff
@@ -110,7 +110,7 @@ int main(int argc, char **argv) {
 
             for (size_t m = 0; m < v1.size(); m++) {
 
-                float absDiff = abs( v1[m]- v2[m]);
+                float absDiff = std::abs( v1[m]- v2[m]);
```

Comment:
Probably need to
```C++
#include <cmath>
```
too...

Reply:
Builds fine so must already be included through some other header. But i can add.

Reply:
> Builds fine so must already be included through some other header.

Probably.  We should not rely on that though.

> But i can add.

Thanks.  I'll merge into master.